packages/flame_tiled/lib/src/renderable_tile_map.dart

```diff
@@ -107,6 +110,74 @@ class RenderableTiledMap {
     return null;
   }
 
+  /// Select a group of tiles from the coordinates [x] and [y].
+  ///
+  /// If [all] is set to true, every renderable tile from the map is collected.
+  ///
+  /// If the [named] or [ids] sets are not empty, any layer with matching
+  /// name or id will have their renderable tiles collected. If the matching
+  /// layer is a group layer, all layers in the group will have their tiles
+  /// collected.
+  TileStack tileStack(
+    int x,
+    int y, {
+    Set<String> named = const <String>{},
+    Set<int> ids = const <int>{},
+    bool all = false,
+  }) {
+    return TileStack._(
+      _tileStack(
+        renderableLayers,
+        x,
+        y,
+        named: named,
+        ids: ids,
+        all: all,
+      ),
+    );
+  }
+
+  /// Recusrive support for [tileStack]
```

Comment:
```suggestion
  /// Recursive support for [tileStack].
```

Reply:
... I need to turn my spellchecker back on.